frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/VerifyMfaCodeHandler.java

```diff
@@ -208,9 +208,9 @@ private void processCodeSession(
             Session session,
             MFAMethodType mfaMethodType,
             APIGatewayProxyRequestEvent input,
-            Context context,
             UserContext userContext,
-            boolean isRegistration) {
+            boolean isRegistration,
+            String clientSessionId) {
```

Comment:
Do you need to add this as a parameter if you've already got the `userContext` object above?